css/style.css

```diff
@@ -241,35 +241,36 @@ button:hover {
 	font-weight: 700;
 	overflow: hidden;
 	height: 40px;
+	display: flex;
 }
 
 .stack h2 input {
 	padding: 0;
-	width: 80%;
+	width: 100%;
 	margin: 0;
 	font-size: 12pt;
 	font-weight: 700;
 	border: 0;
 	background-color: transparent;
-	float: left;
 	min-height: initial;
 }
 
 .stack h2 button,
 .stack .stack-actions {
 	float: right;
+	margin: 3px 1px 3px 19px;
```

Comment:
19px look a bit to stretched. That should be just the same as the margin on the top (inside the grey box).